faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+# On this page (create a table of contents here with anchors):
+
+
+- Which environment should I choose?
+- How do I access RStudio?
+- How can I install custom packages?
+- How do I create a Conda environment?
+- What is a Jupyter kernel?
+- How do I restart my JupyterLab server?
+- Can I recieve announcements about downtime for jupyter.libretexts.org?
+
+## For Instructors
+
+- How do I distribute files to students?
+- How do I set up custom environments for my class?
+
+
+## Which environment should I choose?
+
+After logging in to JupyterHub, you will be taken to a Server Options page where you may select your JupyterLab evironment. This will determine the languages, packages, and kernels available to you. You should continue with whichever option is selected by default unless you have a specific reason to do otherwise.
+
+The New Default Environment is what most users should choose for general JupyterLab use. It is an updated version of what is now called Legacy Default Environment. In this new environment, all supported languages except for Octave and C++ have been upgraded. Depending on the language, some syntax might have changed, so be sure to test your code to see if it works on a newer version. Here's a list of all the languages we support and the version changes:
```

Comment:
```suggestion
The "New Default Environment" is what most users should choose for general JupyterLab use. It is an updated version of what is now called "Legacy Default Environment". In this new environment, all supported languages except for Octave and C++ have been upgraded. Depending on the language, some syntax might have changed, so be sure to test your code to see if it works on a newer version. Here's a list of all the languages we support and the version changes:
```

Reply:
Or use bold or italics for the env names. Something to stand out.

Reply:
Using bold font for these.